.gitignore

```diff
@@ -2,9 +2,12 @@
 **/.terraform/*
 
 # .tfstate files
+.terraform.lock.hcl
```

Comment:
I think we keep this file in git for other projects. It contains the versions of the terraform providers we use, right?

Reply:
Good question! In our own private repos we do check this in. But since repo is meant to be copied by customers, we're deliberately not checking in any of those state files. The `versions.tf` file contains the minimum required versions of each provider.